src/inference/tests/functional/caching_test.cpp

```diff
@@ -2210,6 +2211,7 @@ TEST_P(CachingTest, LoadMulti_race) {
 // Test that it is safe to load multiple devices through loadNetwork
 // In case of sporadic failures - increase 'TEST_DURATION_MS' 100x times for better reproducibility
 TEST_P(CachingTest, LoadMultiWithConfig_race) {
+    GTEST_SKIP();
```

Comment:
REMOVE gtest skip from here! Use `skip_tests_config` instead of this construction